### ISSUE TITLE
Output format variability

### ISSUE THREAD

**Issue by bmaclach** [JacquesCarette/Drasil#2121]
A potential variability we could add to our design language would be the output format.

To start, it should be not be much work to offer a choice between writing output to a file (which we currently do) or writing to standard output. 

More interesting options happen when the output is something that is plottable. The user could choose to visualize the output in plots, though this choice would only make sense if there was list-type output. Potentially a `Plot` choice could be parameterized by the chunks for the quanitities that should go on each axis.

**Comment by smiths** [comment]
I like this idea too.  @JacquesCarette mentioned in a recent comment using some of your ideas for the future work section of your thesis.  I think a list of future design variabilities, like this one, would be right at home in that section.

**Comment by bmaclach** [comment]
Yes, the reason I'm creating these issues is because they are things I thought of while writing that section of my thesis 😄
